README.md

```diff
@@ -43,15 +43,14 @@ can obtain certificates for.
      PackagerBase = 'https://localhost:8080/'
      CertFile = 'path/to/fullchain.pem'
      KeyFile = 'path/to/privkey.pem'
-     OCSPCache = '/tmp/amppkg/ocsp'
+     OCSPCache = '/tmp/amppkg-ocsp'
```

Comment:
Also update testdata/b1/README that has instructions on how to seed the cache. Its location needs to be changed too.

Reply:
Done. Also, it turns out I don't need to update the cert. `--ignore-certificate-errors-spki-list` also ignores the domain name being wrong. But I did anyway so it was consistent. (And found a few issues in doing so.)